homeassistant/components/rfxtrx/binary_sensor.py

```diff
@@ -271,4 +269,5 @@ def apply_cmd(self, cmd):
     def update_state(self, state):
```

Comment:
Will be faster if you mark this as a `@callback` and call `hass.add_job(entity.update_state, state)`

Reply:
(and then use `self.async_write_ha_state()` instead of schedule update)

Reply:
Function is going away in follow-up pulls.